data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/consumer/OrderedConsumerVerticle.java

```diff
@@ -223,4 +249,47 @@ private boolean areAllExecutorsBusy() {
     // All executors are busy
     return true;
   }
+
+  private void recordExecutorQueueLatency(final ConsumerRecordContext recordContext) {
+    final var latency = recordContext.performLatency();
+    logger.debug("Executor queue latency {}", keyValue("latency", latency));
+
+    if (meterRegistry != null) {
+      Metrics
+      .executorQueueLatency(getTags(recordContext))
+      .register(meterRegistry)
+      .record(latency);
+    }
```

Comment:
Since we need to deregister/remove those metrics once a partition is no longer owned I propose to move those to the Executor as in https://github.com/aavarghese/eventing-kafka-broker/pull/7 and remove them as part of the stop process, wdyt?